src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoRecipeWrapper.java

```diff
@@ -118,11 +119,11 @@ public MBPattern(final WorldSceneRenderer sceneRenderer, final List<ItemStack> p
     @SuppressWarnings("NewExpressionSideOnly")
     public MultiblockInfoRecipeWrapper(@Nonnull MultiblockControllerBase controller) {
         this.controller = controller;
-        Set<ItemStackKey> drops = new ObjectOpenHashSet<>();
+        Set<ItemStack> drops = new ObjectOpenCustomHashSet<>(ItemStackHashStrategy.comparingAllButCount());
         this.patterns = controller.getMatchingShapes().stream()
                 .map(it -> initializePattern(it, drops))
                 .toArray(MBPattern[]::new);
-        drops.forEach(stackKey -> allItemStackInputs.add(stackKey.getItemStack()));
+        allItemStackInputs.addAll(drops);
```

Comment:
per your migration guide, should this still provide copies of the `drops` rather than the drops themselves?

Reply:
Gathering the drops creates new ItemStacks for each drop, so they don't need to be copied since they are already all distinct.